pom.xml

```diff
@@ -365,6 +367,56 @@
             </dependencyManagement>
         </profile>
 
+        <profile>
+            <id>to-jdk8</id>
```

Comment:
I like the ranges on these profiles.  The profiles I introduced have gaps.  Can we consolidate these?

Reply:
done